main.py

```diff
@@ -89,6 +90,9 @@ async def on_message(message: Message):
     if message.author == client.user:
         return
 
+    if not isinstance(message.author, Member):
```

Comment:
Could you add a comment to explain why we'd want to exit early in this case? It may not be clear on first glance.